commands/generate.command.ts

```diff
@@ -98,7 +99,7 @@ export class GenerateCommand extends AbstractCommand {
     };
     const table: any = new Table(tableConfig);
     for (const schematic of NestCollection.getSchematics()) {
-      table.push([schematic.name, schematic.alias]);
+      table.push([Colors.green(schematic.name), Colors.cyan(schematic.alias), schematic.description]);
```

Comment:
Can we use the `chalk` library (which is a dependency already) instead of adding a new package?

Reply:
Yes , I will try to implement this using chalk then i will share the result here.